src/Integration/RequestFetcherInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Integration;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * Allows customizing the request information that is attached to the logged event.
+ * An implementation of this interface can be passed to RequestIntegration.
+ */
+interface RequestFetcherInterface
+{
+    public function fetchRequest(): ?ServerRequestInterface;
```

Comment:
Please add a small docblock to explain what this method does. It's kinda repetitive and I'm going to evaluate stopping asking this, but to keep consistency until we decide to change this everything should be documented